### PR TITLE
Adds ADR 005 for choosing the primary cloud provider

### PR DESCRIPTION
**Pull request by cloudify**

### CODE REVIEW

File: docs/architecture/decisions/0005-we-chose-microsoft-azure-as-our-primary-cloud-provider.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Accepted
+
+## Context
+
+As building blocks for the services we're about to develop, we're going to need the following PaaS components:
+
+* A structured data store with automated distribution of data across multiple shards and regions.
+* A message bus that supports partitioning, transactions and non destructive reads (peek).
+* A framework for building _serverless_ applications and APIs.
+* A service for managing push notifications via mobile (both iOS and Apple), and browsers.
```
Comment (gvarisco) [inline docs/architecture/decisions/0005-we-chose-microsoft-azure-as-our-primary-cloud-provider.md]: was it supposed to be "both iOS and Android"?

Comment (gunzip) [inline docs/architecture/decisions/0005-we-chose-microsoft-azure-as-our-primary-cloud-provider.md]: integrated logging and monitoring facilities ?

**Comment by gvarisco** [comment]
We probably shouldn't call it "0005-we-chose-microsoft-azure-as-our-primary-cloud-provider.md" tough.

File: docs/architecture/decisions/0005-we-chose-microsoft-azure-as-our-primary-cloud-provider.md (hunk 1 above)
Comment (cloudify) [inline docs/architecture/decisions/0005-we-chose-microsoft-azure-as-our-primary-cloud-provider.md]: @gvarisco yes sorry

@gunzip yes thanks, will add it

**Comment by cloudify** [comment]
@gvarisco yeah sorry I forgot to change the name of the file before pushing, will remove all comments

**Pull request merged**